browser/js/content.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+function init() {
+  // Only do work on reddit.
+  if (!location.hostname.endsWith('.reddit.com')) return;
```

Comment:
Maybe throw in an `&& location.hostname != 'reddit.com'` to protect ourselves from reddit changing their behavior in the future? Your call.